mne/beamformer/_lcmv.py

```diff
@@ -338,6 +338,100 @@ def apply_lcmv_epochs(epochs, filters, *, return_generator=False,
     return stcs
 
 
+@verbose
+def apply_lcmv_evoked_tfr(evoked_tfr, filters, verbose=None):
+    """Apply Linearly Constrained Minimum Variance (LCMV) beamformer weights.
+
+    Apply Linearly Constrained Minimum Variance (LCMV) beamformer weights
+    on single trial time-frequency data.
+
+    Parameters
+    ----------
+    evoked_tfr : AverageTFR
+        Time-frequency resolved evoked.
```

Comment:
What is an `evoked_tfr`? If it is just a bandpass-filtered or Hilbertized evoked, why do we need a new apply function for this?

Reply:
If anything, I would go with a wrapper, that can use several LCMV filters, and apply them to several filtered evoked objects, and then make a multi-dimensional source object out of it. Or, maybe even better: have a function that can create such a source object from several source objects. i.e., you compute and apply your filters in a for loop, and then glue them together into one `stc`

Reply:
That was my original thought but this leverages existing MNE objects